Gruntfile.coffee

```diff
@@ -68,7 +68,7 @@ module.exports = (grunt) ->
       coverage:
         command: "./node_modules/.bin/istanbul cover ./node_modules/.bin/jasmine-node --captureExceptions test && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
       jasmine:
-        command: "./node_modules/.bin/jasmine-node --verbose --captureExceptions test"
+        command: "./node_modules/.bin/jasmine-node --verbose --captureExceptions --coffee src/spec"
```

Comment:
So jasmine-node can do the check on coffeescript level?
If so, it is using exact the same version of coffeescript compiler we use in the `grunt build` task?